speed-dreams/source-2.2.3/src/libs/robottools/rthumandriver.cpp

```diff
@@ -947,6 +949,13 @@ static void common_drive(const int index, tCarElt* car, tSituation *s)
 #endif
     int scrw, scrh, dummy;
 
+#ifdef RECORD_SESSION
```

Comment:
Dont forget the change comment
```suggestion

// SIMULATED DRIVING ASSISTANCE CHANGE
#ifdef RECORD_SESSION
```